src/main/java/seedu/storemando/logic/commands/SortCommand.java

```diff
@@ -15,8 +15,12 @@
         + "2. " + COMMAND_WORD + " quantity desc\n"
         + "3. " + COMMAND_WORD + " expirydate";
 
-    public static final String MESSAGE_SUCCESS = "Sorted all items";
-
+    public static final String MESSAGE_SUCCESS_QUANTITY_ASC = "Sorted all items based on their quantity in an"
```

Comment:
Maybe you could consider just having this variable declared in the individual subclasses of the SortCommand as they're each used by only one of the subclasses.

Reply:
All right noted!